float32 x_center      # Normalized x-center of the bounding box (0 to 1)
float32 y_center      # Normalized y-center of the bounding box (0 to 1)
float32 width         # Normalized width of the bounding box (0 to 1)
float32 height        # Normalized height of the bounding box (0 to 1)
float32 confidence    # Confidence score (0 to 1)
int32 class_id        # Class ID of the detected object
